### PR TITLE
Specs, docs, and clean up

### PR DESCRIPTION
**Pull request by taylor-work**

### CODE REVIEW

File: cms/README.md [hunk 1]
```diff
@@ -1,3 +1,70 @@
 # Strapi backend
 
 This is the headless CMS backend.
+
+### The following npm scripts are defined
+
+ * `npm run develop`
+    * Runs the cms with file watching turned on. Changes new new content types are detected and strapi is restarted.
```
Comment (danielnaab) [inline cms/README.md:8]:
```suggestion
    * Runs the cms with file watching turned on. Changes and new content types are detected and Strapi is restarted.
```

File: cms/README.md [hunk 2]
```diff
@@ -1,3 +1,70 @@
...
+ * `npm run start`
+    * Runs the cms very similar to production.
+ * `npm run build`
+    * Runs the webpack build for the strapi admin UI. Needed for changes like strapi version changes, plugin changes.
+ * `npm run strapi1`
```
Comment (danielnaab) [inline cms/README.md:13]:
```suggestion
 * `npm run strapi`
```

**Review by danielnaab — APPROVED** [review]
👍  looks great. only thought I had was that if we have tests that modify db state, we might want to be able to isolate it. (not now, later)

**Comment by taylor-work** [comment]
The file `cms/config/env/test/database.json` will handle database mods. Ideally we'd flip it to an in memory sqlite3 db. I'm still working out how to do that with the libraries in play.

**Review by plummer-flex — APPROVED** [review]

**Pull request merged**